examples/bri-3/src/bri/workgroup/workgroups/api/workgroups.controller.spec.ts

```diff
@@ -43,17 +40,16 @@ describe('WorkgroupsController', () => {
       'name',
       'desc',
       BpiSubjectType.External,
-      'publicKey',
+      'pubkey',
```

Comment:
sorry, any specific reasons to rename this?

Reply:
no specific reason.